### ISSUE TITLE
Clickover width settings weird under Bootstrap 2.1

### ISSUE THREAD

**Issue by TimSullivan** [lecar-red/bootstrapx-clickover#13]
In Bootstrap 2.1, both the .popover-inner and the .popover itself need the width set, or things go terribly awry. I hacked a solution by adding 1 line after these:

```
  // set popover's dim's
  this.options.width  && this.tip().find('.popover-inner').width(  this.options.width  );
  this.options.height && this.tip().find('.popover-inner').height( this.options.height );

  // new line
  this.options.width  && this.tip().width(  this.options.width  );
```

**Comment by lecar-red** [comment]
Hi Tim,

Glad you gave it a try on 2.1, i wonder if this will cause a problem with 2.0 bootstrap versions. Any ideas?

Thanks,

Lee

**Comment by TimSullivan** [comment]
I think you need to move forward. Tie your version number to the Bootstrap version number and release an update for 2.1.

**Comment by lecar-red** [comment]
Hi Tim,

Good idea. I'll take a look at adding this next week, With a new version or plunking off current into old version.

**Comment by MiikaL** [comment]
Can confirm that Tim's fix at least appears to work in 2.1.1.

**Comment by lecar-red** [comment]
updated see issue #24
